<ui version="4.0" >
 <class>KeySignatureDialog</class>
 <widget class="QWidget" name="KeySignatureDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>373</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="StaffElementPreviewWidget" native="1" name="preview" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="accidentals" >
         <property name="minimum" >
          <number>-7</number>
         </property>
         <property name="maximum" >
          <number>7</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Region</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="throughBar" >
          <property name="text" >
           <string>Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startBar1" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>through</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="endBar" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="throughEndOfPiece" >
          <property name="text" >
           <string>Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startBar2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>through end of piece</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="toNextChange" >
          <property name="text" >
           <string>Measure</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startBar3" >
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>to next key change</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Staves</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="allStaves" >
        <property name="text" >
         <string>Update all staves</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectedStaff" >
        <property name="text" >
         <string>Only update selected staff</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StaffElementPreviewWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/StaffElementPreviewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>throughBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>startBar1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>74</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel" >
     <x>126</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>throughBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>endBar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>47</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>throughEndOfPiece</sender>
   <signal>toggled(bool)</signal>
   <receiver>startBar2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>118</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toNextChange</sender>
   <signal>toggled(bool)</signal>
   <receiver>startBar3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>72</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel" >
     <x>113</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
